src/components/LicenseRemindModal/index.jsx

```diff
@@ -91,12 +94,19 @@ class LicenseRemindModal extends React.Component {
     if (isBulkRemind) {
       options.bulkRemind = true;
     } else if (!isBulkRemind && user) {
-      options.email = user.emailAddress;
+      options.userId = user.userId;
```

Comment:
Looks like the [`/remind` endpoint in license-manager](https://github.com/edx/license-manager/blob/142abe8de909825644bbfe613014dde7487a78c7/license_manager/apps/api/serializers.py#L88) will take a `user_email` parameter in the POST body so I think we should keep `options.email` here.

I believe the `/revoke` endpoint (currently in code review) also uses `user_email` so we should probably also pass `options.email` in the `LicenseRevokeModal`. (could be done when integrating with the APIs if you'd like)

Reply:
Yes, I think we need to do minor changes like this once we will be integrating FE code with API.